package.json

```
@@ -10,16 +10,18 @@
   "type": "module",
   "scripts": {
     "clean:bundle": "pnpx rimraf dist && pnpx rimraf dist-zip && pnpx turbo clean:bundle",
-    "clean:node_modules": "pnpx turbo daemon stop && pnpx rimraf node_modules && pnpx turbo clean:node_modules",
+    "clean:node_modules": "pnpx turbo daemon stop &&pnpm pnpx rimraf node_modules && pnpx turbo clean:node_modules",
```

```suggestion
    "clean:node_modules": "pnpx turbo daemon stop && pnpx rimraf node_modules && pnpx turbo clean:node_modules",
```

Sth goes wrong here :)